ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/config/DiscoveryConfiguration.java

```diff
@@ -101,7 +101,33 @@
   public static final List<EnodeURL> CLASSIC_BOOTSTRAP_NODES =
       Collections.unmodifiableList(
           Stream.of(
+                  // ETC Cooperative
+                  "enode://8e73168affd8d445edda09c561d607081ca5d7963317caae2702f701eb6546b06948b7f8687a795de576f6a5f33c44828e25a90aa63de18db380a11e660dd06f@159.203.37.80:30303",
```

Comment:
nit: could each sub-group be sorted by client-id?  It makes duplicate checking way easier.  Most IDEs have a sort selected lines option.

Reply:
Not sure what you mean. I just use Sort Lines from VS code to find and/ or remove any duplicates and manually checked them.